### PR TITLE
feat: vscode-application added

### PR DESCRIPTION
**Pull request by NimishKashyap**
### Checklist

- [x] Application license is compatible
- [x] I have created a directory in the form of `apps/<application_name>/<version>`
- [x] I have added the corresponding [metadata](https://docs.napptive.com/catalog/application_metadata.html) file
- [x] I have added the corresponding README file
- [x] I have tested the application in my own namespace and works as expected
- [x] I have tested that the application can be uploaded to my catalog namespace

#### What does this PR do?
Adds VSCode to application catalog integrated with playground CLI.

#### Which as the minimum requirements to launch the application?
This application requires at least 1 GB of PVC storage (can be used with emptyDir storage as well but not recommended), 2 GB of RAM and 1 Core vCPU.

### CODE REVIEW

File: containers/vs-code/v1.82-pg-v6.0.2/steps.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+1. Build dockerfile
+docker build -t nimish8/vscode-server .
```
Comment (aagea) [inline containers/vs-code/v1.82-pg-v6.0.2/steps.txt]:
```suggestion
docker build -t <username>/vscode-server .  
```

File: containers/vs-code/v1.82-pg-v6.0.2/steps.txt [hunk 2]
```diff
@@ -0,0 +1,21 @@
+1. Build dockerfile
+docker build -t nimish8/vscode-server .  
+
+2. Upload dockerfile
+docker push nimish8/vscode-server
```
Comment (aagea) [inline containers/vs-code/v1.82-pg-v6.0.2/steps.txt]:
```suggestion
docker push <username>/vscode-server
```

**Review by aagea — APPROVED** [review]

File: apps/vscode/0.10.1/vscode-oam.yaml [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+        - type: storage
+          properties:
+            pvc:
+              - name: vscode-pvc
+                mountPath: /config
+                resources:
+                  requests:
+                    storage: 1Gi
+      properties:
+        image: nimish8/vscode-server
```
Comment (aagea) [inline apps/vscode/0.10.1/vscode-oam.yaml]: I am going to upload the image to the napptive repo, please change this reference.

**Review by aagea — CHANGES_REQUESTED** [review]

File: apps/vscode/0.10.1/vscode-oam.yaml (hunk 3 above)
Comment (NimishKashyap) [reply apps/vscode/0.10.1/vscode-oam.yaml]: Alright

**Review by aagea — CHANGES_REQUESTED** [review]
Please review your git tree to add in this pull only your changes. Currently, you are including Sonarqube update. Your base branch is behind main.

**Comment by NimishKashyap** [comment]
I've added another PR with git tree fixed #7.

**Pull request closed without merging**